serpent-moltres/isotope-distribution/compare.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def getxs():
+
+    # paramlist = ['DIFFCOEF', 'NSF', 'REMXS', 'CHIT']
+    paramlist = ['NSF', 'REMXS']
+
+    for param in paramlist:
+        inFile = 'xs-a/mhtgr_homoge_' + param + '.txt'
+        with open(inFile, 'r') as i:
+            data = i.readlines()
+        lines = list()
+        for line in data:
+            lines.append(line.split())
+        homogeA = np.array(lines[0][1:], dtype=float)
+
+        inFile = 'xs-b/mhtgr_homoge_' + param + '.txt'
+        with open(inFile, 'r') as i:
+            data = i.readlines()
+        lines = list()
+        for line in data:
+            lines.append(line.split())
+        homogeB = np.array(lines[0][1:], dtype=float)
+
+        rel_err = np.zeros(len(homogeB))
+        for i in range(len(homogeB)):
+            if homogeB[i] == 0:
+                rel_err[i] = 0
+            else:
+                rel_err[i] = np.abs((homogeA[i] - homogeB[i])/homogeB[i])*100
+
+        print(param)
```

Comment:
rm prints

Reply:
These are some of the outputs of the script.